tests/test_examples.py

```diff
@@ -1036,38 +1035,37 @@ def test_print_dot() -> None:
                         rank=same,
                         style=dashed
                 ];
-                "file://{cwl_posix_path}#sorted_output"       [fillcolor="#94DDF4",
+                "sorted_output"       [fillcolor="#94DDF4",
                         label=sorted_output,
                         style=filled];
         }}
-        "file://{cwl_posix_path}#rev" [fillcolor=lightgoldenrodyellow,
+        "rev" [fillcolor=lightgoldenrodyellow,
                 label=rev,
                 style=filled];
-        "file://{cwl_posix_path}#sorted"      [fillcolor=lightgoldenrodyellow,
+        "sorted"      [fillcolor=lightgoldenrodyellow,
                 label=sorted,
                 style=filled];
-        "file://{cwl_posix_path}#rev" -> "file://{cwl_posix_path}#sorted";
-        "file://{cwl_posix_path}#sorted" -> "file://{cwl_posix_path}#sorted_output";
-        "file://{cwl_posix_path}#workflow_input" -> "file://{cwl_posix_path}#rev";
-        "file://{cwl_posix_path}#reverse_sort" -> "file://{cwl_posix_path}#sorted";
+        "rev" -> "sorted";
+        "sorted" -> "sorted_output";
+        "workflow_input" -> "rev";
+        "reverse_sort" -> "sorted";
 }}
-    """.format(
-            cwl_posix_path=cwl_posix_path
-        )
+    """.format()
     )[0]
     stdout = StringIO()
     assert main(["--debug", "--print-dot", cwl_path], stdout=stdout) == 0
     computed_dot = pydot.graph_from_dot_data(stdout.getvalue())[0]
     computed_edges = sorted(
-        (urlparse(source).fragment, urlparse(target).fragment)
+        (source, target)
         for source, target in computed_dot.obj_dict["edges"]
     )
     expected_edges = sorted(
-        (urlparse(source).fragment, urlparse(target).fragment)
+        (source, target) 
```

Comment:
```suggestion
        (source, target)
```

This cause the CI tests to fail. Fixable via `make format`